mods/persistence/modules/mob/new_player.dm

```diff
@@ -86,6 +86,12 @@
 	if(!config.enter_allowed)
 		to_chat(src, SPAN_NOTICE("There is an administrative lock on entering the game!"))
 		return
+
+	switch(alert("Did you create a character?",,"Yes","No"))
```

Comment:
Extraneous , here.

This is also in the wrong place, we should only add this alert if the player doesn't have a player in game, so move it down to the line above create_character() in this same proc.

I'd also prefer it to be something like "Are you sure you want to join the game with the character you've created? This cannot be undone!"

Reply:
I believe the extra `,` means the `alert()` won't have a title.

Reply:
Local man reads ten lines down and realizes someone else said the same thing three weeks ago.